### DISCUSSION TITLE
Changing navbar font color in bootstrap theme

### DISCUSSION THREAD

**Discussion by NickvashKani** [twbs/bootstrap#37651] [Q&A]
Hi, 

I've been wrestling with this issue for hours and was wondering if someone had any ideas how to resolve it. I am using a standard bootstrap theme: jumbotron-narrow. 

https://bootstrapdocs.com/v3.0.2/docs/examples/jumbotron-narrow/

The navbar font color and highlighted page background in the navbar are this bright blue color. I'd like to change it to another color but for the life of me I can't figure out where this color is set. I've even gone so far as to change all the possible colors I think may be related to the nav-bar to no avail. 

Using my browser inspector, the color is #007bff but for the life of me I can't seem to find that value anywhere in the all the files. Does anyone know if there a way to change to navbar color to something other than bright blue? I know it's a minor thing but I've spent a day struggling with this and now I got to see this through :-P.

Thanks 
Nick

**Comment by tbolt** [comment]
I think the most immediate and direct way would be overwriting the bootstrap styles:
```
.nav-pills > li.active > a { 
  background-color: black;
  color: white;
}
```
That's not the most elegant or scalable way though. It's probably best to use the bootstrap customization options:
https://getbootstrap.com/docs/5.2/customize/overview/
https://getbootstrap.com/docs/5.2/customize/color/

It will depend on how you're using bootstrap and what your build system is like but with these options you can change the core theme colors.

Also check out tools where you can create your theme and import `scss/_variables.scss` like https://bootstrap.build/app
